components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/bean/ApplicationRegistrationRequest.java

```diff
@@ -36,6 +36,7 @@
     private String consumerKey = null;
     private String consumerSecret = null;
     private String spTemplateName = null;
+    private Boolean saasApp = null;
```

Comment:
If we set the default value for this to 'false' then we will not need to check for null when consuming this variable.

Reply:
Fixed